src/Drawer/Drawer.js

```diff
@@ -28,42 +29,42 @@ export const styleSheet = createStyleSheet('Drawer', (theme) => {
   };
 });
 
+type Props = {
+  /**
+   * The contents of the `Drawer`
+   */
+  children?: Object,
+  /**
+   * The CSS class name of the root element.
+   */
+  className?: string,
+  /**
+   * If set to true, the drawer will dock itself
+   * and will no longer slide in with an overlay
+   */
+  docked?: boolean,
+  open: boolean,
+  /**
+   * The CSS class name of the paper element.
+   */
+  paperClassName?: string,
+  zDepth: number,
```

Comment:
I don't know docgen at all.  I'm actively working on the flowtype behavior for `defaultProps`...there are some nuances.  I read it as the above should be optional.  There is another place to specify the `type DefaultProps` in the parameterized signature, just prior to `Props` i.e. `<DefaultProps, Props, State>`, so it would be discoverable.

Also note that the babel transform _generates_ props, so this may not be an issue regardless.  See the link in the description above.